region/maridia/outer/Crab Tunnel.json

```diff
@@ -270,6 +194,30 @@
       ],
       "note": "The gate will not spawn in indirect g-mode, and can be freely passed."
     },
+    {
+      "link": [1, 2],
+      "name": "Crab Tunnel Stutter Shinecharge Through The Gate",
```

Comment:
This felt much tighter than any other `canStutterWaterShineCharge`, so I made it notable. We were only able to get it once, so I couldn't really make a good description of when to stutter. I also wasn't sure it was possible in a 2 tile runway.